sunkit_pyvista/plotter.py

```diff
@@ -279,7 +288,61 @@ def plot_field_lines(self, field_lines, **kwargs):
             grid = self._coords_to_xyz(field_line.coords.ravel())
             field_line_mesh = pv.StructuredGrid(grid[:, 0], grid[:, 1], grid[:, 2])
             color = {0: 'black', -1: 'tab:blue', 1: 'tab:red'}.get(field_line.polarity)
+            field_line_mesh.add_field_array([color], 'color')
             self.plotter.add_mesh(field_line_mesh, color=color, **kwargs)
             field_line_meshes.append(field_line_mesh)
 
         self._add_mesh_to_dict(block_name='field_lines', mesh=field_line_meshes)
+
+    def save(self, filepath):
+        """
+        Adds all of the meshes in the current dictionary
+        to a :class:`~pyvista.core.MultiBlock`.
+
+        Parameters
+        ----------
+        filepath : `str` or `pathlib.Path`
+            Name of the file to save as, should have vtm or vtmb as an extension.
+
+        Notes
+        -----
+        This saves the rendered plot as a vtm extended file as well as a directory
+        of the individual meshes with the specified name.
+        """
+        file_path = Path(filepath)
+        directory_path = file_path.with_suffix('')
+        if directory_path.exists():
+            raise ValueError(f"Directory '{directory_path.absolute()}' already exists")
+
+        mesh_block = pv.MultiBlock()
+        for objects in self.all_meshes:
+            for meshes in self.all_meshes[objects]:
+                mesh_block.append(meshes)
+        mesh_block.save(file_path)
+
+    def _loop_through_meshes(self, mesh_block):
+        """
+        Recursively loop to add nested `~pyvista.core.MultiBlock` to the `pyvsita.Plotter`
+        along with the color of the mesh.
+        """
+        for block in mesh_block:
+            if isinstance(block, pv.MultiBlock):
+                self._loop_through_meshes(block)
```

Comment:
I am not a fan of doing this recursively

Reply:
For now, we have a multiblock of multiblocks (containing meshes) for the field lines from pfsspy, the max depth is 2 for field lines but maybe we might have more depth later? I'm not so sure, recursion seemed the best way to do this without knowing the depth